pkg/cloud/services/ec2/launchtemplate.go

```diff
@@ -276,8 +276,8 @@ func updateMachinePoolAnnotation(lts scope.LaunchTemplateScope, annotation, cont
 	lts.GetObjectMeta().SetAnnotations(annotations)
 }
 
-// tagsChanged determines which tags to delete and which to add.
-func tagsChanged(annotation map[string]interface{}, src map[string]string) (bool, map[string]string, map[string]string, map[string]interface{}) {
+// TagsChanged determines which tags to delete and which to add.
+func TagsChanged(annotation map[string]interface{}, src map[string]string) (bool, map[string]string, map[string]string, map[string]interface{}) {
```

Comment:
Unrelated change? Seems not needed to make this public – the function is only used in this file.